### DISCUSSION TITLE
Is it possible to use kitty theme on kitty-diff?

### DISCUSSION THREAD

**Discussion by kacperwyczawski** [kovidgoyal/kitty#6668] [Q&A] [answered]
Is it possible to use kitty theme selected via `kitty +kitten themes` on [kitty-diff](https://sw.kovidgoyal.net/kitty/kittens/diff/)? I want to avoid setting all colors manually, but I can't find anything in the docs. Thanks for help.

**Comment by kovidgoyal** [comment, marked as answer]
No, diff uses a whole bunch of extra colors beyond what a kitty theme
defines.

**Reply by kacperwyczawski** [reply]
Thank you for quick answer.
